phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -116,6 +111,7 @@ public void setOnSelectScript(Consumer<ScriptDTO> onSelectScript) {
      * Show available apps panel
      */
     public void showAvailableApps() {
+        this.showDetailsView(null);
         showRightView(availableApps);
```

Comment:
The name of the method `showRightView` is kind of misleading in this case, because I would expect to set the details view in this case, because it is the most "right" view shown to the user.
I would suggest to rename it to something else.